ggcore/sdk_messages.py

```diff
@@ -1,27 +1,103 @@
 import dataclasses
+import typing
 from dataclasses import dataclass
 
-from ggcore.utils import RequestAuthType
+from ggcore.utils import HttpMethod
 
 
 @dataclass
+class SdkServiceResponse:
+    statusCode: int = None
+    statusText: str = None
+
+    # is a str response here OK or does this need to be more generic/different?
+    response: str = dataclasses.field(default_factory=str)
+
+    exception: Exception = None
```

Comment:
Minor nitpick, but should this be an SdkException type hint rather than base Exception type hint?

Reply:
@HarryDunham I don't think so. The exception here is representing the exception coming back from the http response, so it wouldn't be sdk specific. This exception can be further mapped into a specific sdk exception in an API's request handler.

Reply:
Maybe there is a more specific type hint we could give here though, maybe some http base exception class?

Reply:
After thinking more about this I think it should be a `requests.RequestException`